modules/apps/27-interchain-accounts/host/ibc_module_test.go

```diff
@@ -424,7 +424,7 @@ func (suite *InterchainAccountsTestSuite) TestOnRecvPacket() {
 		{
 			"success with ICA auth module callback failure", func() {
 				suite.chainB.GetSimApp().ICAAuthModule.IBCApp.OnRecvPacket = func(
-					ctx sdk.Context, packet channeltypes.Packet, relayer sdk.AccAddress,
+					ctx sdk.Context, packet channeltypes.Packet, relayer sdk.AccAddress, channelVersion string,
```

Comment:
How come this argument order didn't change?

Reply:
I forgot to update the mock :) Sorting it now for this and all the other places